docs/source/specification.md

```diff
@@ -30,24 +36,24 @@ a dataype (e.g. `ephys, behav, funcimg, histology`) are stored.
 Subject and session folder names cannot contain spaces and must consist of key-value pairs separated
 by underscores, e.g. `sub-001_id-5645332`. 
 
-## Subject
+### Subject
 The subject level is prefixed with the *required* "sub" key - value pair, e.g. `sub-001`. Only one subject
 folder per-subject is permitted, and subject labels must be unique to each subject. *optional* key - value pairs 
 can be added after sub-<label> (e.g. `sub-001_id-5645332`).
 
-## Session
+### Session
 If data for the subject were acquired across multiple sessions,then within the subject 
 directory reside subdirectories named with the *required* prefix "ses", e.g. `ses-<label>`. A session represents a recording, 
 functional imaging or behavioural experimental session, and may contain multiple blocks, or runs, of data acquisition.
 Different sessions may have different combinations of datatypes. *optional* key - value pairs can be added after ses-<label>
 (e.g. `ses-001_date-220516`).
 
-## Datatype
+### Datatype
 Datatypes included at the session level are *required* to have one of the following names `ephys, behav, funcimg, histology`.
```

Comment:
```suggestion
Datatypes are *required* to have one of the following names `ephys, behav, funcimg, histology`.
```
It's a bit confusing the have "at the session level" here, since histology is mentioned. In any case, the required level is clarified by the following sentences.